tests/cmd/fault-trigger/main.go

```diff
@@ -49,16 +43,7 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	mgr := manager.NewManager(kubeProxyImage)
-	hostname, err := util.GetHostname(hostnameOverride)
-	if err != nil {
-		glog.Fatalf("get hostname failed, err: %v", err)
-	}
-	err = mgr.UpdateKubeProxyDaemonset(hostname)
-	if err != nil {
-		glog.Fatalf("update kube-proxy daemonset failed, err: %v", err)
-	}
-
+	mgr := manager.NewManager()
```

Comment:
This cleanup code is copied from https://github.com/pingcap/tidb-operator/pull/503.